src/main/java/codeu/controller/NotificationServlet.java

```diff
@@ -8,6 +8,8 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 import org.mindrot.jbcrypt.BCrypt;
+import java.util.ArrayList;
```

Comment:
nit: these will probably be used later on but it's good to only keep the imports that are needed for this PR